files/scripts/arp_update

```diff
@@ -25,6 +25,8 @@ while /bin/true; do
       fi
   done
 
+  sleep 1
```

Comment:
From your description, I feel the problem here is the ping6 command could get stuck. Adding a wait might not completely solve the problem since there could still be some cases (e.g., the wait time is not enough, some improper vlan config) where it may get stuck. Maybe we can do something like adding a few seconds timeout to `ping6cmd` so that the script would not hang with the command? @prsunny what do you think?

Reply:
Agree with Shi, you can refer line 70 for adding timeout.

Reply:
Added a one second timeout to ping6 command. If we increase the timeout it can lead to a problem when we have a lot of VLAN`s on device. Because each ping to VLAN will be with timeout, now time that we spend on wait is: number of VLAN`s * timeout (by now one second).

Reply:
I think this is OK since the timeout is not expected to wait every time. For example, if the VLANs come up in one second (as the sleep you added in the previous commit), only the first one or two would have the timeout.